Source/SIMPLib/DataContainers/DataContainerArrayProxy.cpp

```diff
@@ -544,3 +544,14 @@ QMap<QString, DataContainerProxy> DataContainerArrayProxy::readMap(QJsonArray js
   }
   return map;
 }
+
+QMap<QString, DataContainerProxy> DataContainerArrayProxy::getdataContainers()
+{
+	return dataContainers;
+}
+
+
+ void DataContainerArrayProxy::setdataContainers(QMap<QString, DataContainerProxy> newDataContainers)
+{
+	 dataContainers = newDataContainers;
```

Comment:
Indentation for goth dataContainer methods. 
Check capitalization on `getdataContainers` and `setdataContainers`.  If that is intentional, it should be marked in both .h and .cpp because it does not look like it is and invites anyone working on the file to "fix" it.